### ISSUE TITLE
Documentation for updating device properties

### ISSUE THREAD

**Issue by crenz** [eclipse/hono#112]
The website currently describes [how to register a device using curl](https://www.eclipse.org/hono/getting-started/) and how to register and unregister devices in general using the [device registration API](https://www.eclipse.org/hono/api/Device-Registration-API/)

I suggest to add the following:

- Extend the device registration API page with information how to update device properties
- Add a curl example to either that page or the getting started page:

```
$ curl -X PUT -i -d "property1=value1&property2=value2" http://localhost:8080/registration/DEFAULT_TENANT/4711
```

**Comment by sysexcontrol** [comment]
IMHO the getting started page should stay as short as possible.

You can find the curl commands already (incl. the PUT) in the documentation of the REST adapter here : https://www.eclipse.org/hono/component/rest-adapter/

My proposal would be:
only extend the Getting started page by a link to the REST Adapter description and a remark that the registration endpoint behaves like a standard REST endpoint (methods POST,GET,PUT,DELETE with standard semantics).
Would that solve the issue?

**Comment by sophokles73** [comment]
Fixed in ec9b735d159b7077ae0f620454cb2eda6e28c196
